Recrutify/Recrutify.Services/Services/CandidateService.cs

```diff
@@ -91,7 +93,10 @@ public async Task<CandidateDTO> CreateAsync(CandidateCreateDTO candidateCreateDT
         {
             var candidate = _mapper.Map<Candidate>(candidateCreateDTO);
             var currentCandidate = await _candidateRepository.GetByEmailAsync(candidate.Email);
+            var currentPrimarySkill = await _primarySkillService.GetAsync(candidateCreateDTO.PrimarySkill.Id);
```

Comment:
throw ValidationException if currentPrimarySkill is null

Reply:
done